### ISSUE TITLE
Leaving the Z-level during malf gibs you.

### ISSUE THREAD

**Issue by Spacemanspark** [ParadiseSS13/Paradise#4154]
**Problem Description**:
Leaving the Z-level during AI Malfunction gibs you.

**What did you expect to happen**: 
It should only gib the AI.

**What happened instead**:
It seems that anyone that leaves the Z-level gets gibbed automatically.

**Why is this bad/What are the consequences:**
I don't think you should get gibbed automatically unless you're the AI.

**Steps to reproduce the problem**:
/>Malf mode.
/>Try to leave the station.
/>Get gibbed.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: It was malfunction and the AI had declared delta. I grabbed a space suit and everything and jumped out the airlock. Immediately upon reaching the end of the Z-level, I was gibbed.
@Fox-McCloud

**Comment by CrAzYPiLoT-SS13** [comment]
GG.

**Comment by Tastyfish** [comment]
kek @Fox-McCloud

**Comment by TheDZD** [comment]
@Fox-McCloud Working as Intended˜.

**Comment by Fox-McCloud** [comment]
Lol. Well then.

**Comment by Spacemanspark** [comment]
Hrm, not sure if multiple people were having the issue.
I was playing Spark, bartender. Hacked into EVA, grabbed a jumpsuit after it called Delta, jumped out through the external access thing in maintenance right next to EVA, and went north. Immediately after it swapped Z-levels, I was gibbed. Just like that. only my posibrain, bloody gibs, and a toolbox that I had used to hack into EVA with remained.

**Comment by Keilden** [comment]
> Hrm, not sure if multiple people were having the issue.
 I was playing Spark, bartender. Hacked into EVA, grabbed a jumpsuit after it called Delta, jumped out through the external access thing in maintenance right next to EVA, and went north. Immediately after it swapped Z-levels, I was gibbed. Just like that. only my posibrain, bloody gibs, and a toolbox that I had used to hack into EVA with remained.
 I attributed it to the malf changes because it was the only thing I could think of. I asked the admins if they did it for some reason and they said they didn't, so I dunno.

Secret admin plot to gib Spark.
